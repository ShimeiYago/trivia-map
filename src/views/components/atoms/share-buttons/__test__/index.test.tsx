import { shallow, ShallowWrapper } from 'enzyme';
import { ShareButtons, Props } from '..';

let wrapper: ShallowWrapper<Props>;

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<ShareButtons url="https://xxx.com" title="xxx" />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
