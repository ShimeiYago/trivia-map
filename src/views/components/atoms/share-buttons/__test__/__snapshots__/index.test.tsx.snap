// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<ForwardRef(Box)
  textAlign="center"
>
  <ForwardRef(ShareButton-facebook)
    title="xxx"
    url="https://xxx.com"
  >
    <Icon
      bgStyle={Object {}}
      borderRadius={0}
      iconFillColor="white"
      round={true}
      size={50}
    />
  </ForwardRef(ShareButton-facebook)>
  <ForwardRef(ShareButton-twitter)
    title="xxx"
    url="https://xxx.com"
  >
    <Icon
      bgStyle={Object {}}
      borderRadius={0}
      iconFillColor="white"
      round={true}
      size={50}
    />
  </ForwardRef(ShareButton-twitter)>
  <ForwardRef(ShareButton-line)
    title="xxx"
    url="https://xxx.com"
  >
    <Icon
      bgStyle={Object {}}
      borderRadius={0}
      iconFillColor="white"
      round={true}
      size={50}
    />
  </ForwardRef(ShareButton-line)>
</ForwardRef(Box)>
`;
